src/lib/Home/Components/Hero/Hero.tsx

```diff
@@ -1,14 +1,34 @@
 import React from 'react';
 import HealButtonLink from '../HealButtonLink/HealButtonLink';
+import Image from 'next/image';
+import hero from './../../Assets/Images/Hero.webp';
+import background from './../../Assets/Images/background.webp';
 
 const Hero: React.FC = () => {
   return (
-    <div data-testid="hero" className="photo-divider text-center p-4">
-      <h1 className="text-2xl font-bold">Hero</h1>
-      <HealButtonLink
-        href="https://askjeeves.com"
-        label="Search for HEAL Data"
-      />
+    <div data-testid="hero" className="hero">
+      <div
+        className="bg-cover bg-center bg-heal-background"
+        style={{ backgroundImage: `url(${background.src})` }}
+      >
+        <div className="container mx-auto px-6">
+          <div className="lg:flex justify-between">
+            <div className="text-white py-44 max-w-xl ">
+              <h1 className="text-4xl font-bold leading-[36px]">
+                The HEAL Data Platform
+              </h1>
+              <p className="my-5 text-2xl font-normal leading-[36px]">
+                is a single web interface which allows visitors to discover,
+                access, and analyze data generated by HEAL Studies.
+              </p>
+              <HealButtonLink href={'#'} label={'Search for Heal Data'} />
```

Comment:
```suggestion
              <HealButtonLink href={'#'} label={'Search for HEAL Data'} />
```
`HEAL` is an abbreviation